File: ios/Classes/Models/HMSNetworkQualityExtension.swift
//
//  HMSNetworkQualityExtension.swift
//  hmssdk_flutter
//
//  Created by Govind on 29/03/22.
//

import Foundation
import HMSSDK

class HMSNetworkQualityExtension {
    static func toDictionary(_ quality: HMSNetworkQuality) -> [String: Any] {
        var dict = [String: Any]()
        dict["quality"]=quality.downlinkQuality
        return dict
    }
}

    

File: android/src/main/kotlin/live/hms/hmssdk_flutter/HMSNetworkQualityExtension.kt
package live.hms.hmssdk_flutter

import live.hms.video.connection.stats.quality.HMSNetworkQuality

class HMSNetworkQualityExtension {

    companion object {
        fun toDictionary(quality: HMSNetworkQuality?): HashMap<String, Any?>? {
            val args = HashMap<String, Any?>()
            if (quality == null) return null
            args["quality"] = quality.downlinkQuality
            return args
        }
    }
}
